src/server/routes/basic-routes.js

```diff
@@ -69,11 +79,12 @@ module.exports = [
             }
 
             // validate username
-            if (uname[0] === '_') {
+            const nameRegex = /[^a-zA-Z0-9][a-zA-Z0-9_\-\(\)\.]*/;
```

Comment:
This should be reverted since some existing usernames don't adhere to this (so they would not be able to reset their password).

Reply:
agreed

Reply:
My mistake - I just realized there was a code fold in the PR which made it look like this was in the reset password endpoint. It is actually in the sign up endpoint so it's all good.